test/tools/isoltest.cpp

```diff
@@ -146,6 +152,16 @@ class TestTool
 
 bool TestTool::m_exitRequested = false;
 
+void TestTool::printOptions(ostream& _stream, string const& _linePrefix, bool) const
+{
+	_stream << _linePrefix << "// ====" << endl;
+	_stream << _linePrefix << "// evmVersion: " << m_options.evmVersion().name() << endl;
+	_stream << _linePrefix << "// optimize: " << m_options.optimize << endl;
+	_stream << _linePrefix << "// useABIEncoderV1: " << m_options.useABIEncoderV1 << endl;
+	_stream << _linePrefix << "// batches: " << m_options.batches << endl;
+	_stream << _linePrefix << "// selectedBatch: " << m_options.selectedBatch << endl;
```

Comment:
The isoltest output is already pretty verbose since it prints the whole source. I'd try to make this more compact.

For example I'd make this look something like: `batch: 3/125` instead having each part on a separate line.

BTW, what are these set to when not using the `--batches` option? Is this `1`/`1` or zeros or something else?

Reply:
I guess it would output something like `0/1` if not using --batches. That's what I got when running isoltest locally with `./test/tools/isoltest --enforce-gas-cost --optimize --no-smt`:
```
...
 // ====
    // evmVersion: london
    // optimize: 1
    // useABIEncoderV1: 0
    // batches: 1
    // selectedBatch: 0
```

Reply:
I think we should add 1 to batch number in that case. I.e. so that's `1/1` by default and so that it's normally from `1/n` to `n/n` rather than from `0/n` to `n-1/n`.